_data/simple-icons.json

```diff
@@ -11484,6 +11484,11 @@
             "hex": "053766",
             "source": "https://sailfishos.org"
         },
+        {
+            "title": "Sails.js",
+            "hex": "14ACC2",
```

Comment:
```suggestion
            "hex": "11ACC2",
```

Reply:
This suggestion makes no sense. This purposed color is not one of their brand colors.

Reply:
It's the colour used in the sails of the EPS logo. It's only a minor difference from yours, so could just be my software interpreting the EPS differently.